pdp/handlers_upload.go

```diff
@@ -497,7 +497,7 @@ func (p *PDPService) handleFindPiece(w http.ResponseWriter, r *http.Request) {
 	}
 
 	response := struct {
-		PieceCID string `json:"piece_cid"`
+		PieceCID string `json:"pieceCid"`
 	}{
 		PieceCID: pieceCid.String(),
 	}
```

Comment:
Rather than define this as an anonymous struct, could it be defined as a named struct? Ideally along with all the other request response type curio returns? 

This could allow Storacha, and other services depending on Curio, to import the types directly, rather than manual coordination.

Reply:
I do agree with your ask to have proper names structs all over the place but Curio team is prioritising the market v2 work instead of enhancements to existing implementation. You can open a PR which can be reviewed by both Curio and FilOz if you are interested in contributing this change. Otherwise, Market PR is likely to be merged in next 2 weeks.